bundletester/spec.py

```diff
@@ -311,6 +311,8 @@ def filter_yamls(yamls):
 
     result = []
     for yamlfn in yamls:
+        if yamlfn.endswith('metrics.yaml'):
+            continue
```

Comment:
@ktsakalozos Right. I just mean that we should probably make the check:

```
if yamlfn == 'metrics.yaml':
    ...
```
It's actually not that big of a deal, though. I'm mainly good to merge this.